### ISSUE TITLE
bugs in jtable demo download (not part of this repo, but...)

### ISSUE THREAD

**Issue by kenirwin** [hikalkan/jtable#1846]
Hi folks,

I just downloaded jtable and the demos from jtable.org. I found a couple of things I'd like to fix in the demos, but they are not part of this github repo. Is there a different repo for the demos so I can help contribute to those? 

Thanks!

**Comment by hikalkan** [comment]
Demos and web site are not under Github. If it's simple, please write here. Thanks a lot.

**Comment by kenirwin** [comment]
Yes, the fixes are simple. In the html HEAD tag on both jTableSimple.php and jTableSimplePagedSorted.php files in the PHP demos, the paths to the first and jtable.css and jquery.table.js links uses a capital S in 'Scripts' instead of 'scripts' in the path. On my system, this results in broken links. If you make the filepaths all lower-case 's', that fixes the problem.
```
<link href="Scripts/jtable/themes/lightcolor/blue/jtable.css" rel="stylesheet" type="text/css" />
<script src="scripts/jquery-1.6.4.min.js" type="text/javascript"></script>
<script src="scripts/jquery-ui-1.8.16.custom.min.js" type="text/javascript"></script>
<script src="Scripts/jtable/jquery.jtable.js" type="text/javascript"></script>
```

Thanks!

**Comment by kenirwin** [comment]
PS -- this is a great little jquery library. Thanks for sharing it!

**Comment by hikalkan** [comment]
Thanks I will fix it soon.

**Comment by erdarshandoshi** [comment]
One another thing I found while working with jquery is 
when there are 2 tables one is parent and one is child table
Earlier problem was child table was not getting toggled
So i fix that part by using below code
  //Open child table when user clicks the image
                            $img.click(function () {
                                var parentTable = $("#StudentTableContainer1");
                                
                                var tr = $(this).parents("tr"),
                                 isChildRowOpen = parentTable.jtable("isChildRowOpen", tr);
                           
                                if (isChildRowOpen) {
                                    $(parentTable.jtable("getChildRow", tr)).slideUp();
                                    return;
                                }

But PROBLEM which is still not resolved is when i click on second row child table-->First row child table is not getting closed. :(
